resources/templates/mustache/preprod/govuk-template.mustache.html

```diff
@@ -869,8 +869,22 @@ <h2>
         {{/sessionTimeout}}
 
         {{#inlineScript}}
-            {{{inlineScript}}}
+            {{inlineScript}}
```

Comment:
Removed a bracket here, was this intentional?

Reply:
Yes, on the back of Kurt's comments I removed the extra brackets as well so it cannot be escaped.

Reply:
After discussing with you I've added the brackets back in